.gitlab-ci.yml

```diff
@@ -1,8 +1,19 @@
 image: node:16
 
 variables:
-  CARGO_HOME: ${CI_PROJECT_DIR}/.cargo/
   SCCACHE_DIR: ${CI_PROJECT_DIR}/.cache/sccache
+  NODE_OPTIONS: --max-http-header-size=128000
+  # Postgres
+  POSTGRES_DB: app_db
+  POSTGRES_USER: app_user
+  POSTGRES_PASSWORD: password
```

Comment:
Is there a reason to name here `POSTGRES_PASSWORD` and use it as `PGPASSWORD` afterwards?

Reply:
POSTGRES_PASSWORD is used by postgres container and PGPASSWORD by psql